fs-storage/src/utils.rs

```diff
@@ -52,6 +54,38 @@ where
     Ok(data)
 }
 
+pub fn remove_files_not_in_ram<K, V>(
```

Comment:
This "clean up" function is only relevant to `folder_storage` right now and can be kept in that module itself. Later if it becomes relevant to other storages it can be moved here.